### ISSUE TITLE
Deploy python package via Anaconda

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-moco#506]
I created an Anaconda Cloud account for opensim-org. https://anaconda.org/opensim-org/dashboard

We should make CI create/upload the Python packages.

**Comment by moorepants** [comment]
It'd be great to see it in conda forge.

**Comment by chrisdembia** [comment]
Hey @moorepants ! That’s the long term plan but I can probably set up a package using our own CI in two days, which I don’t think I could do for conda forge. Also, I hope that Moco will be part of opensim-core, and then a separate package wouldn’t be necessary.

**Comment by moorepants** [comment]
I see. Let us know if we can help with conda forge packages when you get there.

**Comment by chrisdembia** [comment]
Okay, will do. :)

**Comment by chrisdembia** [comment]
@moorepants , Moco depends on an unreleased fork of `opensim-core`. Would this cause issues for conda-forge? Ideally, we would not create a separate `opensim-core` package for Moco.

**Comment by moorepants** [comment]
I think that conda-forge allows package maintainers to release rc, beta, alpha candidates. So you could do that. In general, they package only released software. You can add patches to released software but it shouldn't modify any public API.

**Comment by chrisdembia** [comment]
Okay thanks that’s helpful.

**Comment by chrisdembia** [comment]
I'm running into some difficulty using conda locally to build Moco. I don't know how to access more recent compilers.

**Comment by moorepants** [comment]
Yes, there are all these new ways to specify compilers: https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html
